src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class AbstractFigure implements Figure, Draw {
+    protected final Colors color;
+
+    public AbstractFigure(Colors color) {
+        this.color = color;
+    }
+
+    @Override
+    public abstract double calculateArea();
+
+    @Override
+    public abstract void draw();
```

Comment:
Abstract class does not need to override interface methods - that's the duty of subclasses. Remove this.